ocp_resources/observability_addon.py

```diff
@@ -0,0 +1,9 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class ObservabilityAddon(NamespacedResource):
```

Comment:
Missing call to super and implementation on to_dict

Reply:
I wish to use this resource for view only after the observability is enabled.
Should we implement a to_dict anyway?

Reply:
Yes, this is not only for your personal use, this should be full resource implementation